packages/web-components/src/wallet-connection/querySwingsetParams.js

```diff
@@ -0,0 +1,18 @@
+import { QueryClient, createProtobufRpcClient } from '@cosmjs/stargate';
+import { QueryClientImpl } from '@agoric/cosmic-proto/swingset/query.js';
+
+/** @typedef {import('@agoric/cosmic-proto/dist/agoric/swingset/query').QueryParamsResponse} QueryParamsResponse */
```

Comment:
it may be worth sometime soon getting all the `.js` in this repo over to `.ts`, to have a consistent syntax and the ability to use real imports before TS 5.5